sys-devel/llvm/llvm-17.0.6.ebuild

```diff
@@ -51,7 +51,6 @@ BDEPEND="
 	sys-devel/gnuconfig
 	kernel_Darwin? (
 		<sys-libs/libcxx-${LLVM_VERSION}.9999
-		>=sys-devel/binutils-apple-5.1
```

Comment:
Does this mean you don't need any, or that you specifically don't want this linker?  Should you depend on native-cctools instead?

Reply:
No. This was required with gcc toolchain. But with llvm toolchain this is not only not necessary, but actively breaks the setup. When using llvm tools themselves, everything works fine, but if binutils-apple is involved, it will crash in runtime at some point. See also `packages` file in https://github.com/gentoo/gentoo/pull/35042

native-cctools are not necessary anymore after the bootstrap is finished, and stage3 is up and running - at that point only clang+llvm from the prefix are used, but not Xcode compilers/tools, with the exception of still using headers from `${EPREFIX}/MacOSX.sdk`

Reply:
The linker used in the final system is ld64.lld, and not the host's one. This is also handled via profile at https://github.com/gentoo/gentoo/pull/35042

Reply:
So ld64.lld comes from a separate package, or as part of llvm and is not managed by binutils-config?  If so, should be fine.

Reply:
It's from sys-devel/lld